packages/list-reusable-blocks/src/components/import-form/index.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { withInstanceId } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { Button, Notice } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import importReusableBlock from '../../utils/import';
+
+class ImportForm extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			isLoading: false,
+			error: null,
+			file: null,
+		};
+
+		this.isMounted = true;
```

Comment:
Re: Previous comment, this is actually a part of the React component API, though discouraged, which main explain errors around attempts to set it.

https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html

Reply:
I know the antipattern but sometimes I'm pragmatic and don't want to introduce more complex solutions like a cancelable promise or something like that. Especially since it's a small JS file for a separate page here.

Reply:
The point was less about it being discouraged / deprecated, and more to the point that this is already a property defined on a default React component, and we're overriding it.